src/i_sound.c

```diff
@@ -51,6 +52,14 @@ static music_modules_t music_modules[] =
     { &music_opl_module, 1 },
 };
 
+static music_module_t *persistent_modules[] =
```

Comment:
What does "persistent" mean in this context?

Reply:
These modules are always loaded, so we can test many music formats on every song change.

Reply:
Good idea, could you please put this into a comment?

Reply:
I removed `persistent_modules`, now `music_oal_module` acts like SDL_Mixer before.

Reply:
Looks very clean now!